projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -734,6 +742,62 @@ function wpcom_add_new_page_check( $post_id, $post ) {
 }
 add_action( 'wp_insert_post', 'wpcom_add_new_page_check', 10, 3 );
 
+/**
+ * Determine `edit_page` task visibility. The task is visible if there is at least one page and the add_new_page task is complete.
+ *
+ * @return bool True if we should show the task, false otherwise.
+ */
+function wpcom_is_edit_page_task_visible() {
+	if ( ! wpcom_is_task_option_completed( array( 'id' => 'add_new_page' ) ) ) {
+		return false;
+	}
+
+	return ! empty(
+		get_posts(
+			array(
+				'numberposts' => 1,
+				'post_type'   => 'page',
+			)
+		)
+	);
+}
+
+/**
+ * When a page is updated, check to see if it's the About page and mark the task complete accordingly.
```

Comment:
Nit: Looks like this comment needs to be updated since we're not specifically checking an "About" page now. :)